types/mempool/priority_nonce_test.go

```
@@ -970,3 +970,41 @@ func TestNextSenderTx_TxReplacement(t *testing.T) {
+	}
+
+	require.Equal(t, 4, mp.CountTx())
+
+	orderedTxs := fetchTxs(mp.Select(ctx, nil), 100000)
+	require.Equal(t, len(txs), len(orderedTxs))
+
+	// check order
+	for i, tx := range orderedTxs {
+		require.Equal(t, txs[i].id, tx.(testTx).id)
+	}
+
```

golangci-lint (1.62.2)

File is not `gofumpt`-ed with `-extra`

(gofumpt)